index.html

```diff
@@ -8,7 +8,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 </head>
 <body>
-  <div id="app">Loading...</div>
+  <main id="app">Loading...</main>
```

Comment:
For the CSS – changed the selector from `.app` to `main` so app styling doesn't rely on class names being present.